frontend/components/forms/validation.ts

```diff
@@ -0,0 +1,72 @@
+import { useIntl } from 'react-intl';
```

Comment:
Don't forget to move this file. 

Suggestions: https://vizzuality.slack.com/archives/C037B9T1PT2/p1648116992394859?thread_ts=1648112907.252129&cid=C037B9T1PT2